index.html

```diff
@@ -752,7 +752,7 @@ <h1>Speakers</h1>
 											NodeSchool is an open source project run by volunteers with two goals: to create high quality programming curriculum and to host community learning events.
 										</div>
 										<div class="social-links vertical-align">
-											<a href="#"><i class="icon fa fa-external-link"></i></a>
+											<a href="https://soareschen.com"><i class="icon fa fa-external-link"></i></a>
 											<a href="https://github.com/soareschen"><i class="icon fa fa-github"></i></a>
```

Comment:
This link seems to be dead. Change `https` to `http` and it ll work.

Reply:
Oke, Wait...